examples/ndn-ping/README.md

```diff
@@ -0,0 +1,49 @@
+# ndn-ping
+
+This application demonstrates the usage of the package ndn-riot.
+This example basically enable the user to setup a ndn data server, and a ndn client that can request the data.
+Any board with a default netdev can be used to run this example.
+
+# Setting up for native
+
+You need to create `tap` and `tapbr` devices using RIOT's `tapsetup` script before stating the application:
```

Comment:
Below an affirmative style is used, I would do the same here as well: `Create tap and tapbr, etc`